MANIFEST.in

```diff
@@ -1,3 +1,9 @@
 include LICENSE
 include README.md
 include CHANGELOG.md
+include CODEOWNERS
```

Comment:
I don't think we need to include the CODEOWNERS file.

And on a side note, please feel free to add yourself to this file.  Ideally we could get it set up where a review from EITHER you or me is good.